Makefile

```diff
@@ -26,6 +26,13 @@ lint:
 	@find . -name "*.py" | xargs pylint -f parseable
 	@echo
 
+shortlint:
+	@echo
+	@echo "******************************** Lint *****************************************"
+	@echo
+	@find . -name "*.py" | xargs pylint -f parseable -E
```

Comment:
Jenkins doesn't use `make`.  It has its own independant syntax and build plans.

We might want to change that at some stage, but for now we're fine.